api/pom.xml

```diff
@@ -27,4 +27,132 @@
+                    <bottom>
+<![CDATA[
+        Copyright &#169; ${inceptionYear} &#x2018; {currentYear} {organizationName}. All rights reserved.
+        <br>
+        Use is subject to <a href="{@docRoot}/doc-files/license.html" target="_top">license terms</a>.
+    ]]>
+                    </bottom>
+                    <tags>
+                        <tag>
+                            <name>implSpec</name>
+                            <placement>a</placement>
+                            <head>Implementation Specification:</head>
+                        </tag>
+                    </tags>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>attach-javadocs</id>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+                <version>4.2.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>bnd-process</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <Bundle-SymbolicName>org.eclipse.microprofile.context</Bundle-SymbolicName>
+                    <Bundle-Name>MicroProfile Context Propagation bundle</Bundle-Name>
+                    <archive>
+                        <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
+                    </archive>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <profiles>
+        <profile>
+            <id>eclipse-jarsigner</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.eclipse.cbi.maven.plugins</groupId>
+                        <artifactId>eclipse-jarsigner-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>sign</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-baseline-maven-plugin</artifactId>
+                <version>4.2.0</version>
+                <configuration>
+                    <base>
+                        <version>1.4</version>
```

Comment:
This should be 1.0

Reply:
Thanks - it is fixed in commit https://github.com/eclipse/microprofile-context-propagation/pull/207/commits/515eaaa1bed6e3e9c49725750668b45a9ee4e1ea